msibi/tutorials/propane/propane.ipynb

```diff
@@ -9,7 +9,7 @@
     "Created by Davy Yue 2017-06-14\n",
     "\n",
     "\n",
-    "### Import packages\n",
+    "### Imports\n",
     "\n",
     "According to PEP8 standards, imports are grouped into \n",
     "1. standard library imports \n",
```

Comment:
No need to list pep8 standards in this notebook. The contents of this cell can just be:
"Imports"